rel/overlay/etc/default.ini

```diff
@@ -494,6 +494,8 @@ partitioned||* = true
 ;retries_per_request = 5
 ; Use checkpoints
 ;use_checkpoints = true
+; Attempt to use bulk_get for fetching documents from the source
+;use_bulk_get = true
```

Comment:
I suppose another approach would be to change the default to `false` for now to keep existing behavior, but switch it to `true` for a major version change?

Reply:
Since there is a fallback already, in most cases it should work, and this would be for more exceptions or in rare situations.